test/test.ml

```diff
@@ -1,23 +1,26 @@
 type ava
-type exe
+type t
 
-external t:ava = "ava" [@@bs.module]
+external ava: ava = "ava" [@@bs.module]
```

Comment:
We need `let ava = require("ava")` in generated Js. How can I achieve that with treating it as a Js.t? Thanks!